GCP/legos/gcp_list_vms_access/gcp_list_vms_access.py

```diff
@@ -19,15 +19,15 @@ class InputSchema(BaseModel):
 
 
 @beartype
-def gcp_list_VMs_access_printer(output):
+def gcp_list_vms_access_printer(output):
```

Comment:
Why didn't the old names work @jayasimha-raghavan-unskript ?
Is it because the JSON file had different entry name in it?

Reply:
Please add the assert, or test case to make this diff complete...